tests/test-webapps/test-servlet-spec/test-spec-webapp/src/main/java/com/acme/test/MultiPartTest.java

```diff
@@ -116,8 +116,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             throw new ServletException(e);
         }
     }
-    
 
-  
-   
+    public static void copy(InputStream in,
+                            OutputStream out)
+        throws IOException
+    {
+        int bufferSize = 8192;
+        byte buffer[] = new byte[bufferSize];
+
+        while (true)
+        {
+            int len=in.read(buffer,0,bufferSize);
+            if (len<0 )
+                break;
+            out.write(buffer,0,len);
+        }
+    }
```

Comment:
What's the reason to inline `IO.copy()`?

Reply:
because you wouldn't let me do a release of a fixed jetty-util jar :)
Once we have a fixed 9.4 jetty-util release, we can re-instate the tests that use it and then we can remove this inline (I told you it was a pain!)
I don't want to bring in jetty-10 util just for this 1 usage.